target/linux/x86/config-5.10

```diff
@@ -512,15 +403,14 @@ CONFIG_X86_MCE_AMD=y
 # CONFIG_X86_MCE_INJECT is not set
 CONFIG_X86_MCE_INTEL=y
 CONFIG_X86_MCE_THRESHOLD=y
-CONFIG_X86_MINIMUM_CPU_FAMILY=5
+CONFIG_X86_MINIMUM_CPU_FAMILY=6
```

Comment:
This will be inherited by legacy subtarget which could break it for some CPUs. As mentioned use the legacy subtarget to refresh this config.

Reply:
Noted. Could you confirm if your legacy device worked with the 5.4 kernel?

It's hard to see how the previous `X86_MINIMUM_CPU_FAMILY=5` would reliably work on an i386 CPU. In fact, Linux kernels for a while now explicitly state:
```
Note that the 386 is no longer supported, this includes
AMD/Cyrix/Intel 386DX/DXL/SL/SLC/SX, Cyrix/TI 486DLC/DLC2,
UMC 486SX-S and the NexGen Nx586.
```

Reply:
To clarify, i386/486 CPU's haven't been supported since 89878f60f47abf998446c5813e476fe9f0a8a38c, which is noted as causing "kconfig havoc". The potential x86/legacy problems can be resolved by explicitly setting the CPU type and refreshing the subtarget.

Reply:
> To clarify, i386/486 CPU's haven't been supported since [89878f6](https://github.com/openwrt/openwrt/commit/89878f60f47abf998446c5813e476fe9f0a8a38c), which is noted as causing "kconfig havoc". The potential x86/legacy problems can be resolved by explicitly setting the CPU type and refreshing the subtarget.

That is correct, but there are some VIA C3 CPUs (some mini ITX boards) which don't support cmov instruction and by selecting 6 family we are disabling them.